src/eko/anomalous_dimensions/__init__.py

```diff
@@ -80,7 +81,7 @@ def gamma_ns(order, mode, n, nf):
     ----------
         order : int
             perturbative order
-        mode : "m" | "p" | "v"
+        mode : "-" | "+" | "V"
```

Comment:
Since it's written above that mode has to be a number (specifically `u2`), then it can not be `"-" | "+" | "V"`.